loops-and-arrays.js

```diff
@@ -12,11 +12,19 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
+let log = "";
 for (let i = 0; i < names[0].length; i++) {
-
-    console.log(names[i] + "je");
+    log = [names[0] + "je", names[1] + "je", names[2] + "je", names[3] + "je"];
+    }
+{
+    console.log(log);
```

Comment:
De opdracht is dat je met je script de console.log(names) aanpast, hier maak je een ander variabele en je logt daarna dat variabele.. niet de oorspronkelijke opdracht, ik zie dat je ze hierna wel allemaal goed hebt!